projects/ladspa.org/package.yml

```diff
@@ -0,0 +1,42 @@
+distributable:
+  url: https://www.ladspa.org/download/ladspa_sdk_{{version.raw}}.tgz
+  strip-components: 1
+
+versions:
+  url: https://www.ladspa.org/download/index.html
+  match: /ladspa_sdk_\d+\.\d+\.tgz/
+  strip:
+    - /^ladspa_sdk_/
+    - /\.tgz$/
+
+platforms:
+  - linux
+
+dependencies:
+  github.com/libsndfile/libsndfile: ^1.2
+
+runtime:
+  env:
+    LADSPA_PATH: ${{prefix}}/lib/ladspa:$LADSPA_PATH
```

Comment:
are we sure this variable supports colon separation?

Reply:
> are we sure this variable supports colon separation?

I think yes https://archive.flossmanuals.net/pure-data/list-of-objects/hosting-plugins.html